web-client/src/views/Dashboards/DashboardPetitioner.jsx

```diff
@@ -27,6 +28,7 @@ export const DashboardPetitioner = connect(
               {dashboardExternalHelper.showCaseList && <CaseListPetitioner />}
             </div>
             <div className="tablet:grid-col-4">
+              {dashboardExternalHelper.showCaseSearch && <CaseSearchBox />}
```

Comment:
Because this is the `DashboardPetitioner` page, would `showCaseSearch` ever be false? Do we need a check at all?

Reply:
removed showCaseSearch from helper and the three caselist components